CHANGELOG.md

```diff
@@ -3,7 +3,9 @@
 ### vNEXT
 - Client exposes new asyncronous middleware to modify `OperationOptions` [PR #78](https://github.com/apollographql/subscriptions-transport-ws/pull/78)
```

Comment:
It will be good to also mention those PR's here:

https://github.com/apollographql/subscriptions-transport-ws/pull/135
https://github.com/apollographql/subscriptions-transport-ws/pull/137

I believe they are very important and also will be merged. I'm also preparing a third PR to give some refactoring in the changes introduced by PR #78